packages/vite-plugin/package.json

```diff
@@ -36,12 +36,13 @@
     "@types/node": "^20.17.57",
     "playwright": "^1.52.0",
     "tsup": "^8.0.0",
-    "vite": "^6.3.4",
+    "vite": "^7.1.5",
     "vitest": "^3.0.0"
   },
   "dependencies": {
     "@netlify/dev": "4.5.9",
-    "@netlify/dev-utils": "^4.1.3"
+    "@netlify/dev-utils": "^4.1.3",
+    "dedent": "^1.7.0"
```

Comment:
26 KB, zero dependencies, maintained: https://node-modules.dev/#install=dedent

worth it for proper syntax highlighting and syntax parsing in IDEs